include/z64camera.h

```diff
@@ -95,18 +95,18 @@
 
 // Camera stateFlags. Variety of generic flags
 #define CAM_STATE_0 (1 << 0) // Must be set for the camera to change settings based on the bg surface
-#define CAM_STATE_1 (1 << 1) // Must be set for Camera_UpdateWater to run
-#define CAM_STATE_2 (1 << 2)
+#define CAM_STATE_CHECK_WATER (1 << 1) // Must be set for Camera_UpdateWater to run
+#define CAM_STATE_2 (1 << 2) //  Must be set for the camera to change settings based on the bg surface
 #define CAM_STATE_3 (1 << 3) // Customizable flag for different functions
-#define CAM_STATE_4 (1 << 4)
-#define CAM_STATE_5 (1 << 5)
-#define CAM_STATE_6 (1 << 6)
+#define CAM_STATE_4 (1 << 4) // Related to respawn mode in player
+#define CAM_STATE_5 (1 << 5) // Prevents camera mode from changing mode depending on flags
+#define CAM_STATE_DISTORTION (1 << 6) // Set when camera distortion is on
 #define CAM_STATE_7 (1 << 7) // Set in play, unused
-#define CAM_STATE_8 (1 << 8) // Camera (eye) is underwater
-#define CAM_STATE_9 (1 << 9)
+#define CAM_STATE_UNDERWATER (1 << 8) // Camera (eye) is underwater
+#define CAM_STATE_9 (1 << 9) // Related to water
```

Comment:
this kinda looks like "camera updatewater has initialized" but idk, it's weird

for example the code under `bgCamIndexBeforeUnderwater < 0` is dead code?

Reply:
I looked into this and the difference between states 8 and 9 was that one block in `Camera_UpdateWater` deals entirely with the player position (`CAM_STATE_9`) while the other block deals entirely with the camera position/eye (`CAM_STATE_8`). So they've been labeled and docs as such and docs/comments were added inside `Camera_UpdateWater`